airflow_config/scripts/init_airflow_service.py

```diff
@@ -0,0 +1,181 @@
+    if os.getenv("AIRFLOW__CORE__EXECUTOR") == "CeleryExecutor":
+        AIRFLOW__CELERY__RESULT_BACKEND = f"db+postgresql://{AIRFLOW_DB_USER}:{AIRFLOW_DB_PWD}@{AIRFLOW_DB_HOST}:{AIRFLOW_DB_PORT}/{AIRFLOW_DB}"
+        os.environ["AIRFLOW__CELERY__RESULT_BACKEND"] = AIRFLOW__CELERY__RESULT_BACKEND
+
+# CeleryExecutor drives the need for a Celery broker, here Redis is used
+if os.getenv("AIRFLOW__CORE__EXECUTOR") == "CeleryExecutor":
+    # Check if the user has provided explicit Airflow configuration concerning the broker
+    if os.getenv("AIRFLOW__CELERY__BROKER_URL") is None:
+        # Default values corresponding to the default compose files
+        REDIS_PROTO = os.getenv("REDIS_PROTO", "redis://")
+        REDIS_HOST = os.getenv("REDIS_HOST", "redis")
+        REDIS_PORT = os.getenv("REDIS_PORT", "6379")
+        REDIS_PASSWORD = os.getenv("REDIS_PASSWORD", "")
+        REDIS_DBNUM = os.getenv("REDIS_DBNUM", "1")
+
+    # When Redis is secured by basic auth, it does not handle the username part of basic auth, only a token
+    if REDIS_PASSWORD is not None:
+        REDIS_PREFIX = f":{REDIS_PASSWORD}@"
+    else:
+        REDIS_PREFIX = None
+
+    AIRFLOW__CELERY__BROKER_URL = (
+        f"{REDIS_PROTO}{REDIS_PREFIX}{REDIS_HOST}:{REDIS_PORT}/{REDIS_DBNUM}"
+    )
+    os.environ["AIRFLOW__CELERY__BROKER_URL"] = AIRFLOW__CELERY__BROKER_URL
+
+# Make cornflow connection for response from workers
+if os.getenv("AIRFLOW_CONN_CF_URI") is None:
+    # Default values corresponding to the default compose files
+    CORNFLOW_HOST = os.getenv("CORNFLOW_HOST", "cornflow")
+    CORNFLOW_PORT = os.getenv("CORNFLOW_PORT", "5000")
+    os.environ["CORNFLOW_HOST"] = CORNFLOW_HOST
+    os.environ["CORNFLOW_PORT"] = CORNFLOW_PORT
+    # Make the uri connection to get back response from airflow
+    AIRFLOW_CONN_CF_URI = f"cornflow://{CORNFLOW_SERVICE_USER}:{CORNFLOW_SERVICE_PWD}@{CORNFLOW_HOST}:{CORNFLOW_PORT}"
+    os.environ["AIRFLOW_CONN_CF_URI"] = AIRFLOW_CONN_CF_URI
+
+# Check LDAP parameters for active directory
+if os.getenv("AIRFLOW_LDAP_ENABLE") == "True":
+    # Rename webserver config file for using LDAP
+    os.rename(
+        f"{AIRFLOW_HOME}/webserver_ldap.py", f"{AIRFLOW_HOME}/webserver_config.py"
+    )
+    # Update LDAP env values
+    os.environ["AIRFLOW_LDAP_URI"] = os.getenv("AIRFLOW_LDAP_URI", "ldap://openldap:389")
+    os.environ["AIRFLOW_LDAP_SEARCH"] = os.getenv("AIRFLOW_LDAP_SEARCH", "ou=users,dc=example,dc=org")
+    os.environ["AIRFLOW_LDAP_BIND_USER"] = os.getenv(
+        "AIRFLOW_LDAP_BIND_USER", "cn=admin,dc=example,dc=org"
+    )
+    os.environ["AIRFLOW_LDAP_BIND_PASSWORD"] = os.getenv("AIRFLOW_LDAP_BIND_PASSWORD", "admin")
+    os.environ["AIRFLOW_LDAP_UID_FIELD"] = os.getenv("AIRFLOW_LDAP_UID_FIELD", "cn")
+    os.environ["AIRFLOW_LDAP_ROLE_MAPPING_ADMIN"] = os.getenv(
+        "AIRFLOW_LDAP_ROLE_MAPPING_ADMIN",
+        "cn=administrators,ou=groups,dc=example,dc=org",
+    )
```

Comment:
can we do the same thing here with the tuples of environment variables values / names and the for loop like above?